src/e2e/global-teardown.ts

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import https from "https";
+import { readFileSync, writeFileSync } from "fs";
+import { CustomTestResult } from "../../E2EReporter";
+
+async function globalTearDown() {
+  // check env for cron, teardown is only needed for teardown run
+  if (process.env.IS_CRON) {
+    // get data from e2e-reports branch files -- these are api calls
+    const {
+      existingFailedReports,
+      existingFlakeReports,
+      existingFullReports,
+      newFullReportJson,
+      newFailReportJson,
+      newFlakeReportJson,
```

Comment:
nit: but keeps the `existing*` and `new*` in the same order.
```suggestion
      newFailReportJson,
      newFlakeReportJson,
      newFullReportJson,
```